HTTPShortcuts/app/src/main/kotlin/ch/rmy/android/http_shortcuts/variables/types/DateType.kt

```diff
@@ -30,15 +30,15 @@ constructor(
         if (variable.rememberValue) {
             variablesRepository.setVariableValue(variable.id, DATE_FORMAT.format(selectedDate))
         }
-        return SimpleDateFormat(getDateFormat(variable), Locale.US)
+        return SimpleDateFormat(getDateFormat(variable), Locale.getDefault())
             .format(Date.from(selectedDate.atStartOfDay(ZoneId.systemDefault()).toInstant()))
     }
 
     private fun getInitialDate(previousValue: String?): LocalDate =
         previousValue
             ?.let {
                 try {
-                    LocalDate.parse(it, DateTimeFormatter.ofPattern(DEFAULT_FORMAT, Locale.US))
+                    LocalDate.parse(it, DateTimeFormatter.ofPattern(DEFAULT_FORMAT, Locale.getDefault()))
```

Comment:
Changing the locale here might lead to problems, as this is the pattern used to restore a remembered date from storage. It's an internal format that should never change, as otherwise previously stored values might fail to be restored.

(Actually, this line should have been making use of `DATE_FORMAT` to begin with, rather than reimplementing the `DateFormatter`.)